src/neo/Ledger/Blockchain.cs

```diff
@@ -411,23 +492,18 @@ private static ImmutableHashSet<UInt160> UpdateExtensibleWitnessWhiteList(Protoc
 
     internal class BlockchainMailbox : PriorityMailbox
     {
-        public BlockchainMailbox(Akka.Actor.Settings settings, Config config)
+        public BlockchainMailbox(Settings settings, Config config)
             : base(settings, config)
         {
         }
 
         internal protected override bool IsHighPriority(object message)
         {
-            switch (message)
+            return message switch
             {
-                case Header[] _:
-                case Block _:
-                case ExtensiblePayload _:
-                case Terminated _:
-                    return true;
-                default:
-                    return false;
-            }
+                Header[] or Block or ExtensiblePayload or Terminated => true,
+                _ => false,
+            };
```

Comment:
return without switch ( message is Header[] || message is Block ||...?

Reply:
No difference.